redis_api/api/routes.py

```diff
@@ -21,10 +22,11 @@ def get_total_contributions():
     :return: int representing the total for (either all time or current election)
     """
     try:
-        response = r.execute_command("JSON.GET", "TotalContributions")
-        if not response:
-            return empty_response("TotalContributions")
-        return jsonify({"TotalContributions": json.loads(response)})
+        # TODO re-add real data later
+        # response = r.execute_command("JSON.GET", "TotalContributions")
+        # if not response:
+        #     return empty_response("TotalContributions")
+        return jsonify({"TotalContributions": 100000})
```

Comment:
Looks good to me, but it might be more flexible to have this work based on a command-line flag provided when starting the server.  In other words, if the user includes a flag (i.e. "fake_data"), then all the API's will return fake data instead of reading from Redis.

No changes required to submit, this can always come later.

Reply:
That's a good idea, I'll take time to implement it later. I think for now, the candidates endpoint does have real data but the rest of the endpoints don't hence all the commented out code. Though maybe I can pass in arguments to the flag so one can specify which endpoints they'd like fake data served for.